www/js/controller.js

```diff
@@ -865,7 +865,7 @@ define([
 				console.log("Parser error: ", err);
 				console.log(err.message);
 				console.log(err.Error);
-				this.setAuthorStatus(this.currentID,'equation','incorrect');
+				this._model.authored.setAuthorStatus(this.currentID,'equation','incorrect');
```

Comment:
If it is too much right now we can actually make a card for this right now. I think we will anyway have to move some part of the code from con-author to controller.js when we make con-student, as that is the idea. Only the difference of the two will sit in specific files.